### ISSUE TITLE
Error After Loading App Into Team

### ISSUE THREAD

**Issue by jocrabtree** [OfficeDev/microsoft-teams-apps-requestateam#138]
I'm getting this error after I load the app into the Team:

![image](https://user-images.githubusercontent.com/51301604/91328879-1b247300-e78d-11ea-8d0e-64fd6354e76a.png)

I thought it might be due to the incorrect admin group id, but I have verified I'm using the correct one several times. Also, I'm not sure where this error is being thrown from (Power Apps, Flow, Sharepoint?). I'm guessing it's from the PowerApps side where I input the admin group id of the Team, but I'm not entirely sure.

I'd appreciate any help that can be provided.

**Comment by richwright** [comment]
Follow the these steps. 
[https://github.com/OfficeDev/microsoft-teams-apps-requestateam/wiki/Deployment-Guide#step-2-create-admin-group](https://github.com/OfficeDev/microsoft-teams-apps-requestateam/wiki/Deployment-Guide#step-2-create-admin-group)

**Comment by jocrabtree** [comment]
@richwright - Thank you for your help. I see what the issue was now and have fixed it on my end. On Step 3 - Part 8 the 'OnStart' function looked like this for me after creation:

If(CountRows(Filter(Office365Groups.ListGroupMembers("groupid").value, id = Office365Users.MyProfile().Id)) > 0,Set(showadmin,true),Set(showadmin,false)); **Set(gblManagedPath, LookUp('Team Request Settings',Title="SPOManagedPath").Value)**

I modified to this:
If(CountRows(Filter(Office365Groups.ListGroupMembers("groupid").value, id = Office365Users.MyProfile().Id)) > 0,Set(showadmin,true),Set(showadmin,false));

NOTE: I replaced the groupid with the actual groupid for my team within Teams.

It seems to be working for me now. Thanks again for the assist!
